lib/saluki-event/src/metric/value/mod.rs

```diff
@@ -470,6 +470,19 @@ impl MetricValues {
             Self::Distribution(_) => "distribution",
         }
     }
+
+    /// Returns whether the metric is a serie.
+    pub fn is_serie(&self) -> bool {
+        matches!(
+            self,
+            Self::Counter(_) | Self::Rate(_, _) | Self::Gauge(_) | Self::Set(_)
+        )
+    }
+
+    /// Returns whether the metric is a sketch.
+    pub fn is_sketch(&self) -> bool {
+        matches!(self, Self::Histogram(_) | Self::Distribution(_))
```

Comment:
Does this match the Agent behavior? I'm trying to think through this one because by default, we would materialize aggregates over the histogram and send them as gauges... but if we set histograms to be copied to distributions, then we would turn them into sketches, and so on... 🤔

Reply:
Oops, looks like its a [serie](https://github.com/DataDog/datadog-agent/blob/main/pkg/metrics/histogram.go#L156). Updated